pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/GrpcBrokerRequestHandler.java

```diff
@@ -59,6 +62,7 @@ public GrpcBrokerRequestHandler(PinotConfiguration config, BrokerRoutingManager
       AccessControlFactory accessControlFactory, QueryQuotaManager queryQuotaManager, TableCache tableCache,
       BrokerMetrics brokerMetrics, TlsConfig tlsConfig) {
     super(config, routingManager, accessControlFactory, queryQuotaManager, tableCache, brokerMetrics);
+    LOGGER.info("Starting Grpc BrokerRequestHandler.");
```

Comment:
Could be misleading here ? Put it under start() ?

Reply:
it was originally before constructor. let me re-word it as "using GRPC bro...` since the purpose it to indicate what request handler is being constructed.